src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -371,21 +333,10 @@ export const VenueLandingPage: React.FC = () => {
           </div>
         </div>
       </div>
-
-      {user && selectedEvent && (
-        <PaymentModal
-          selectedEvent={selectedEvent}
-          show={isPaymentModalOpen}
-          onHide={closePaymentModal}
-          setEventPaidSuccessfully={setEventPaidSuccessfully}
-          eventPaidSuccessfully={eventPaidSuccessfully}
-        />
-      )}
-
       <AuthenticationModal
         show={isAuthenticationModalOpen}
         onHide={closeAuthenticationModal}
-        afterUserIsLoggedIn={() => setShouldOpenPaymentModal(true)}
+        afterUserIsLoggedIn={() => {}}
```

Comment:
Is `afterUserIsLoggedIn` an optional prop? If so, we don't need to pass it anymore at all. If it isn't optional, then please make it optional + ensure to check if it's defined in `AuthenticationModal` before trying to call `afterUserIsLoggedIn` there. If there is nowhere left in the codebase that actually passes anything meaningful to `afterUserIsLoggedIn`, then perhaps we can remove it entirely.

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40